pwiz_tools/Skyline/TestConnected/UnifiFunctionalTest.cs

```diff
@@ -80,11 +98,30 @@ protected override void DoTest()
             var importResultsDlg = ShowDialog<ImportResultsDlg>(SkylineWindow.ImportResults);
             var openDataSourceDialog = ShowDialog<OpenDataSourceDialog>(importResultsDlg.OkDialog);
             var editAccountDlg = ShowDialog<EditRemoteAccountDlg>(() => openDataSourceDialog.CurrentDirectory = RemoteUrl.EMPTY);
-            RunUI(()=>editAccountDlg.SetRemoteAccount(_testAccount));
+
+            // Test invalid server URLs
+            RunUI(() => editAccountDlg.SetRemoteAccount(_testAccount.ChangeServerUrl("localhost")));
+            AssertAlertDlgContainsMessage(() => editAccountDlg.TestSettings(), ToolsUIResources.EditRemoteAccountDlg_ValidateValues_Invalid_server_URL_);
+            RunUI(() => editAccountDlg.SetRemoteAccount(_testAccount.ChangeServerUrl("https://localhost:12345"))); // resolves, but no server there
+            AssertAlertDlgContainsMessage(() => editAccountDlg.TestSettings(), "Unable to connect to the remote server");
```

Comment:
I think I tested and these pass on Japanese but I don't understand why. These errors come from the system though rather than Skyline. Perhaps they are only localized if the entire OS is set to that language?